.github/workflows/test-pull-request.yml

```diff
@@ -23,4 +23,4 @@ jobs:
         run: xcodebuild -scheme UID2 -sdk iphonesimulator16.2 -destination "OS=16.2,name=iPhone 14"
 
       - name: Run unit tests
-        run: xcodebuild test -scheme UID2Tests -sdk iphonesimulator16.2 -destination "OS=16.2,name=iPhone 14"
\ No newline at end of file
+        run: xcodebuild test -scheme UID2Tests -sdk iphonesimulator16.2 -destination "OS=16.2,name=iPhone 14"
```

Comment:
I know this is not your change, but please update the checkout@v3 to @v4 to remove the warning